containers/server-postgresql-image/Dockerfile

```diff
@@ -0,0 +1,31 @@
+# SPDX-License-Identifier: MIT
+#!BuildTag: uyuni/server-postgresql:latest
+
+ARG BASE=registry.opensuse.org/opensuse/postgres:16
+# or ARG BASE=registry.suse.com/suse/postgres:16
+FROM $BASE
+
+COPY --chmod=0555 uyuni-postgres-config.sh /docker-entrypoint-initdb.d/uyuni-postgres-config.sh
+COPY --chmod=0555 uyuni-entrypoint.sh /usr/local/bin/uyuni-entrypoint.sh
+
+# LABELs
+ARG PRODUCT=Uyuni
+ARG VENDOR="Uyuni project"
+ARG URL="https://www.uyuni-project.org/"
+ARG REFERENCE_PREFIX="registry.opensuse.org/uyuni"
+
+# Build Service required labels
+# labelprefix=org.opensuse.uyuni.server-postgresql
+LABEL org.opencontainers.image.name=server-postgresql-image
+LABEL org.opencontainers.image.title="${PRODUCT} Postgresql container"
+LABEL org.opencontainers.image.description="${PRODUCT} Postgresql image"
+LABEL org.opencontainers.image.created="%BUILDTIME%"
+LABEL org.opencontainers.image.vendor="${VENDOR}"
+LABEL org.opencontainers.image.url="${URL}"
+LABEL org.opencontainers.image.version=5.1.1
```

Comment:
```suggestion
LABEL org.opencontainers.image.version=5.1.0
```

Reply:
This is going to be bumped at each beta / RC release too